reference/tools/microsoft/helpers.rst

```diff
@@ -54,6 +54,8 @@ is_msvc_static_runtime
 conan.tools.microsoft.subsystems
 ==================================
 
+.. _conan_tools_microsoft_unix_path:
```

Comment:
Reminder: review the `Autotools.install()` docstring to add the information about the usage of `unix_path(DESTDIR)` by default.